internal/oci/uvm.go

```diff
@@ -345,6 +346,22 @@ func SpecToUVMCreateOpts(ctx context.Context, s *specs.Spec, id, owner string) (
 		wopts.NoDirectMap = ParseAnnotationsBool(ctx, s.Annotations, annotations.VSMBNoDirectMap, wopts.NoDirectMap)
 		wopts.NoInheritHostTimezone = ParseAnnotationsBool(ctx, s.Annotations, annotations.NoInheritHostTimezone, wopts.NoInheritHostTimezone)
 		wopts.AdditionalRegistryKeys = append(wopts.AdditionalRegistryKeys, parseAdditionalRegistryValues(ctx, s.Annotations)...)
+		isolationType := ParseAnnotationsString(s.Annotations, annotations.WCOWIsolationType, "")
+
+		if isolationType != "" {
+			if isolationType == "SNP" {
+				wopts.IsolationType = "SecureNestedPaging"
+			} else if isolationType == "VBS" {
+				wopts.IsolationType = "VirtualizationBasedSecurity"
+			} else if isolationType == "GuestStateOnly"{
+				wopts.IsolationType = "GuestStateOnly"
+			} else {
+				return nil, fmt.Errorf("invalid WCOW isolation type %q", isolationType)
+			}
+		}
+
+		wopts.DisableSecureBoot = ParseAnnotationsBool(ctx, s.Annotations, annotations.WCOWDisableSecureBoot, false)
```

Comment:
We have a `handleWCOWSecurityPolicy` function that handles CWCOW specific annotations (it already handles some of these annotations). Can you move this there? 

Also, let's not fail if the user sets the isolation type as "VirtualizationBasedSecurity" & "SecureNestedPaging" instead of "VBS"/"SNP".